ansible_collections/arista/avd/roles/eos_designs/doc/network-services-v4.0.md

```diff
@@ -318,12 +318,14 @@ svi_profiles:
             ip_address_virtual: < IPv4_address/Mask >
 
         # List of L3 interfaces | Optional.
-        # This will create IP routed interface inside VRF. Length of interfaces, nodes and ip_addresses must match.
+        # This will create IP routed interface inside VRF. Length of interfaces, nodes and ip_addresses and descriptions (if used) must match.
         l3_interfaces:
           - interfaces: [ <interface_name1>, <interface_name2>, <interface_name3> ]
             ip_addresses: [ <IPv4_address/Mask>, <IPv4_address/Mask>, <IPv4_address/Mask> ]
             nodes: [ < node_1 >, < node_2 >, < node_1 > ]
             description: < description >
+            # `descriptions` has precedence over `description`
+            descriptions: [ <description1>, <description2>, description3> ]
```

Comment:
can't we keep a single key here "description" which can be a string or list , same as we are doing in custom_structured_configuration_prefix ?

Reply:
the problem is that the key name would not make sense - `interfaceS` `ip_addressES` `nodeS` ... singular `description` would be confusing